copy/multiple.go

```diff
@@ -140,7 +151,49 @@ func (c *copier) copyMultipleImages(ctx context.Context, policyContext *signatur
 				UpdateDigest:    updatedManifestDigest,
 				UpdateSize:      int64(len(updatedManifest)),
 				UpdateMediaType: updatedManifestType})
-		default:
+		}
+		if instance.op == instanceCopyClone {
+			processedOp = false
+			c.Printf("Replicating image %s (%d/%d)\n", instance.sourceDigest, i+1, len(instanceCopyList))
+			c.Printf("Replicating image %s (%d/%d)\n", instance.sourceDigest, i+1, len(instanceCopyList))
+			unparsedInstance := image.UnparsedInstance(c.rawSource, &instanceCopyList[i].sourceDigest)
+			src, err := image.FromUnparsedImage(ctx, options.SourceCtx, unparsedInstance)
+			if err != nil {
+				return nil, fmt.Errorf("initializing image from source %s: %w", transports.ImageName(c.rawSource.Reference()), err)
+			}
+			imgConfig, err := src.OCIConfig(ctx)
+			if err != nil {
+				return nil, fmt.Errorf("parsing image configuration: %w", err)
+			}
+			imgPlatform := imgspecv1.Platform{Architecture: imgConfig.Architecture,
+				OS:         imgConfig.OS,
+				OSVersion:  imgConfig.OSVersion,
+				OSFeatures: imgConfig.OSFeatures,
+				Variant:    imgConfig.Variant}
+			addAnnotations := make(map[string]string)
+			if instance.forceZstd {
```

Comment:
BTW _eventually_ we should support `forceZstd` also for a non-clone `instanceCopyCopy`, to implement “take an existing gzip image, and replace all instances with Zstd ones, not copying the gzip variants at all”.

That doesn’t need to exist _today_. But it might be possible structure the code in a way that makes it trivially cheap already in this PR, so it’s something to consider in the background.

(And if this PR doesn’t implement `instanceCopyCopy, forceZstd = true`, the `forceZstd` field should have a comment saying that it only works for `…CopyClone`.)